README.md

```diff
@@ -5,6 +5,12 @@ Example automations for using Sessy with Home Assistant
 Controls Sessy to maintain value X on the grid meter.
 - Load balancing across multiple Sessy's
 - Except certain load via offset entities (e.g. EV charger)
+- Some attention points:
+  - Make sure you don't have any helper entities with `_power_strategy`, `_power_setpoint`, `_state_of_charge` or `_system_state` in the name
+  - `Update Interval` needs to be slower than the grid connection sensor update interval.
+  - A reasonable `timeout` value is 5 seconds
+  - `Smoothing` default of 100 is a good starting point. This is the maximum step adjustment. When your sensor updates more frequent you should lower the smoothing value. A large value may result in instable control 
```

Comment:
The optimal smoothing value varies based on the speed of your grid meter. A lower smoothing value means it takes longer to react to changes, a higher value means the reaction is faster. Fluctuations normally only appear if the automation is acting on outdated information, which is mostly when the set update interval is lower than the update interval of the grid meter.
For my setup with a DSMR5 P1 grid meter, a smoothing value of 1000 works fine at an update interval of 5 seconds.